index.html

```diff
@@ -1104,7 +1104,12 @@
 recognized. Note that sites can do harm even if they can't be completely certain
 that visits come from the same person, so [=user agents=] should also take steps
 to prevent such probabilistic recognition. The [[[Privacy-Threat]]] discusses
-the tradeoffs involved.
+the tradeoffs involved. Note that if a [=user agent=] blocks [=recognition=]
+between different sites but doesn't notice when a single site contains multiple
+[=contexts=], this can encourage [=actors=] to bundle multiple contexts into a
+single site and privilege the [=actors=] who can do so. These are both harmful,
+but that harm is likely less than the harm from allowing [=cross-site
```

Comment:
Likely more? If user agents block cross-domain recognition, but not cross-context recognition within domains, then single-context domain owners are pressured to either (1) sell out to the owner of a multi-context domain, leading to the formation of large, systemically risky, firms or (2) enter into an agreement to proxy their traffic through a server hosting a multi-context domain, creating an unnecessary single point of failure for multiple web services, along with risk of cross-organization personal info sharing.

Or we can state that both cross-context/same-domain recognition and cross-domain recognition are bad without rating one as worse than the other?

Reply:
The change I made for Robin also avoids comparing the harms.

Reply:
Thank you--made another comment there (this is tricky, may people will read as being encouraged to consolidate more contexts on fewer domains)